app/views/admins/contacts/index.html.erb

```diff
@@ -0,0 +1,69 @@
+<% provide(:page_title, t('title.contacts')) %>
+
+<div class="row">
+  <div class="col-12">
+
+    <div class="row">
+      <div class="col-md-8 col-sm-12">
+        <div class="input-group mb-3">
+          <div class="input-group">
+            <input type="text" class="form-control" placeholder="Procurar...">
+            <span class="input-group-append">
+              <button class="btn btn-outline-info" type="button"><i class="fe fe-search"></i></button>
+            </span>
+          </div>
+        </div>
+      </div>
+
+      <div class="col-md-4 col-sm-12">
+        <%= link_to new_admins_contact_path, class: 'btn btn-outline-primary btn-block' do %>
+          <i class="fe fe-plus mr-2"></i><%= t('title.new_contact') %>
+        <% end %>
+      </div>
+
+    </div>
+
+    <div class="row">
+      <div class="col-12">
+        <div class="table-responsive">
+          <table class="table table-hover table-outline table-vcenter text-nowrap card-table">
+            <thead>
+              <tr>
+                <th><%= t('activerecord.attributes.attribute_commons.name') %></th>
+                <th><%= t('activerecord.attributes.attribute_commons.email') %></th>
+                <th><%= t('activerecord.attributes.contact.institution') %></th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody>
+              <% @contacts.each do | contact |  %>
+                <tr>
+                  <td><%= contact.name %></td>
+                  <td><%= contact.email %></td>
+                  <td><%= contact.institution.acronym %></td>
+                  <td>
+                    <%= link_to admins_contact_path(contact), class: 'text-info', title: 'Detalhes' do %>
+                      <i class="fe fe-search"></i>
```

Comment:
Use <%= icon('fas', 'user-cog', class: 'mr-2') %> for font-awesome icons. Review all code.